cli/toolchain_cmd.go

```diff
@@ -604,24 +604,24 @@ Refusing to install Basic toolchain because %s is not installed or is not on the
 	return nil
 }
 
-func cloneToolchain(dest, toolchain string) error {
-	if fi, err := os.Stat(dest); os.IsNotExist(err) {
+func cloneToolchain(destDir, cloneURI string) error {
+	if fi, err := os.Stat(destDir); os.IsNotExist(err) {
 		// Clone
-		if err := os.MkdirAll(filepath.Dir(dest), 0700); err != nil {
+		if err := os.MkdirAll(filepath.Dir(destDir), 0700); err != nil {
 			return err
 		}
 
-		cmd := exec.Command("git", "clone", toolchain)
-		cmd.Dir = filepath.Dir(dest)
+		cmd := exec.Command("git", "clone", cloneURI)
+		cmd.Dir = filepath.Dir(destDir)
 		cmd.Stdout = os.Stdout
 		cmd.Stderr = os.Stderr
 		return cmd.Run()
 	} else if err != nil {
 		return err
 	} else if !fi.Mode().IsDir() {
-		return fmt.Errorf("not a directory: %s", dest)
+		return fmt.Errorf("not a directory: %s", destDir)
 	}
-	log.Printf("Toolchain directory %q already exists, using existing version.", dest)
+	log.Printf("Toolchain directory %q already exists, using existing version.", destDir)
```

Comment:
I wanted to see if the readability of this `cloneToolchain` func could be improved by following [Indent Error Flow style suggestion](https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow):

```Go
func cloneToolchain(destDir, cloneURI string) error {
	switch fi, err := os.Stat(destDir); {
	case err != nil && !os.IsNotExist(err):
		return err
	case err == nil && !fi.Mode().IsDir():
		return fmt.Errorf("not a directory: %s", destDir)
	case err == nil && fi.Mode().IsDir():
		log.Printf("Toolchain directory %q already exists, using existing version.", destDir)
		return nil
	}

	// Clone.
	...
}
```

It does look easier to read/reason about to me. Feel free to take this if you want. Completely optional; I just wanted to share this because the thought occurred to me.